src/comments.ts

```diff
@@ -106,8 +106,7 @@ export const OfferSelfMerge = deletedWhenNotPresent("merge-offer", tag =>
             |> Ready to merge
             |
             |and I'll merge this PR almost instantly. Thanks for helping out! :heart:
-            |${otherOwners.length === 0 ? "" : `
-            |(${otherOwners.map(o => "@" + o).join(", ")}: you can do this too.)`}`}));
+            |${otherOwners.length === 0 ? "" : `(Any owner can do this too.)`}`}));
```

Comment:
Instead of pinging nobody, this should probably ping those reviewers which are also owners.

Reply:
After the first mention in `PingReviewers`, every owner is already subscribed and will receive a notification for each commit and comment.

I think this is easier to reason about if you think `@user` as a `Subscribe @user to this PR` rather than `Ping @user`. Do you want to _subscribe_ them over and over?? That's the problematic part.

There should be one subscription, and then GitHub takes care of successive notifications already.

Reply:
I disagree that it is "over and over" to remind everyone to actually merge the PR.

Reply:
Mentions are not reminders, they're subscriptions. GitHub **already** reminds you after the subscription. A mention offers no improvement over _no mention_ if you're already subscribed.

Reply:
I don't fully agree. Explicit mentions are delivered to a different email address than subscriptions, a fact that some people use to filter their inboxes. Not to mention the notification page, which splits them apart:

![image](https://github.com/DefinitelyTyped/dt-mergebot/assets/5341706/c073272b-2e03-4f83-bf61-bad659a8793b)

So, I think that there is still nuance here, moreso than the simple "is or isn't subscribed". Someone could reasonably use "mentioned" as being a higher priority than "subscribed".

But, I also don't have a strong opinion. But, I also have not seen anyone complain about this whole thing outside of your linked issue either.

Reply:
Please believe when I say that **mentions are subscriptions**.

> Explicit mentions are delivered to a different email address than subscriptions

No.

Look at these two emails: one has a mention, the other is a followup notification. They have identical to/cc fields.

Further mentions or lack of mentions do no change that, unless it's a new "subscription" reason, like "assigned" or "review requested", in which case _that_ sticks around.

<table>
<tr>
 <td valign=top><img src="https://github.com/DefinitelyTyped/dt-mergebot/assets/1402241/10f45ff7-6c8c-4c0a-a602-9709d2e5fc79" width="320">
 <td valign=top> <img src="https://github.com/DefinitelyTyped/dt-mergebot/assets/1402241/5c3fe76d-57ce-490c-b5d0-7a0f4dcebdf1" width="320">
</table>


> Not to mention the notification page, which splits them apart

No. Same exact thing here. Once you've been mentioned, new notifications will continue to appear in the "Mentioned" filter.

… because, once again, GitHub thinks of "subscription reason", not "notification reason"



> But, I also don't have a strong opinion

💙 

> I also have not seen anyone complain

It looks me months to find this bot. Most people will just block the repo or just [asked to be removed](https://github.com/DefinitelyTyped/DefinitelyTyped/pulls?q=is%3Apr+is%3Aclosed+label%3AUnowned) altogether.

Heck I opened this issue in April 2023 and we're only discussing it now, which option is easier for people? 😃

Reply:
> No. Same exact thing here. Once you've been mentioned, new notifications will continue to appear in the "Mentioned" filter.

From one of your pings:

![Screenshot_20240127-045531](https://github.com/DefinitelyTyped/dt-mergebot/assets/5341706/e46686b5-11e9-4f3a-abff-179ba948311b)

Then the email for the above message:

![Screenshot_20240127-045708](https://github.com/DefinitelyTyped/dt-mergebot/assets/5341706/8bb50d58-2957-4300-b949-f65d490b8c09)

Notice no "mentioned" the second time. It's possible that the merge event behaves inconsistently.

> Most people will just block the repo or just [asked to be removed](https://github.com/DefinitelyTyped/DefinitelyTyped/pulls?q=is%3Apr+is%3Aclosed+label%3AUnowned) altogether.

That's the wrong logical direction; maybe all annoyed people remove themselves, but not all people who remove themselves do so due to pings specifically.

Reply:
@jakebailey did you unsubscribe from this thread between January 19 and January 26? I just posted screenshots that show the opposite behavior, so either you unsubscribed, or the behavior isn't really reliable.

Either way the question is simple, I won't continue to debate it:

- do you want to bother people who clicked Unsubscribe?

Because that's not great netiquette, also technically violating CAN-SPAM

Reply:
> did you unsubscribe from this thread between January 19 and January 26?

No, I did not. Thinking about it, though

Reply:
To be clear, I am not against the change, but just trying to explain that there is nuance in a workflow change like this.

I'm just not really understanding the antagonism here 😞